Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -77,11 +77,11 @@ void EnhancementsWidget::CreateWidgets()
 
   m_pp_effect = new ToolTipComboBox();
   m_configure_pp_effect = new QPushButton(tr("Configure"));
+  m_force_texture_filtering = new GraphicsChoice({tr("Default"), tr("Force"), tr("None")},
+                                                 Config::GFX_ENHANCE_FORCE_FILTERING);
```

Comment:
MayImilae already commented on this but I also think the names used here are pretty poor.

Reply:
Yeah. I personally have no issue with the current names but changing to "Force Trilinear/Nearest Neighbor" seems to be the nicest way of doing this (in my opinion).

Reply:
For what it's worth "Default", "Force Trilinear/Bilinear", and "Force Nearest Neighbor" would make sense to me.